CDTDatastore/Encryption/CDTEncryptionKey.m

```diff
@@ -32,7 +32,13 @@ - (NSData *)data {
 }
 
 #pragma mark - Init object
-- (instancetype)init { return [self initWithData:nil]; }
+- (instancetype)init
+{
+#pragma clang diagnostic push
```

Comment:
Wouldn't it be easier and clearer to just implement this as `return nil;` as it's marked unavailable?